<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandaProcess</class>
 <widget class="QDialog" name="RandaProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RandaProcess</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>                                                                雷达信号智能辨识系统</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string> 加载雷达数据文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadDZTButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="CurrentFile">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>  当前文件：</string>
        </property>
       </widget>
       <widget class="QLabel" name="loadDZTfile">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="processButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>开始识别</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="CurrentPro">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>当前进度：</string>
          </property>
         </widget>
         <widget class="QLabel" name="CurrentOP">
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar { border: 2px solid grey; border-radius: 5px; background-color: #FFFFFF;}QProgressBar::chunk { background-color: #55ffff; width: 10px;}QProgressBar { border: 2px solid grey; border-radius: 5px; text-align: center;}
</string>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QPushButton" name="saveProImageButton">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>保存处理后雷达图片</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="saveImageButton">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>保存未处理雷达图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="proReportButton">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
         </property>
         <property name="text">
          <string>生成处理报告</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
